--- flags.py.orig	2020-08-21 11:51:41.839259563 +0000
+++ flags.py	2020-08-03 06:38:12.733197333 +0000
@@ -127,3 +127,9 @@
         'frr-version', LooseVersion, default=DEFAULT_ZSERV_FRR_VERSION,
         help='FRRouting version when integrated with FRRouting (e.g., 3.0)'),
 ], group='zapi')
+
+CONF.register_cli_opts([
+    # k_shortest_forwarding
+    cfg.IntOpt('k-paths', default=1, help='number for k shortest paths'),
+    cfg.StrOpt('weight', default='hop',
+               help='weight type of computing shortest path.')])
